client/src/common/Expandable.jsx

```diff
@@ -153,13 +157,15 @@ export default class Expandable extends React.Component {
       styleContent,
       content,
       borderRadius,
+      styleArrow,
+      styleArrowFocus,
     } = this.props
     const {open, display, focusing} = this.state
 
     const arrow = showArrow ? open ? (
-      <span>&nbsp;&#9660;</span>
+      <span>&nbsp;&#9660;&nbsp;</span>
```

Comment:
We should probably get away from using &nbsp; at all, rather than adding it. I think the reason it originally made sense on the left was becuase it was inline with text and would adhere to the letterspacing. I don't think that's the case anymore as we are wrapping the arrow in it's own distinct div/styles. It seems like this is an attempt to add padding to the right of the arrow, which isn't good practice to accomplish through an invisible characters -- especially if we have a mechanism to add this padding through existing styles.

Reply:
That's a good idea. I was trying to get the padding between the arrow icon and the focus outline to look balanced, and it would probably make more sense to remove the leading nbsp and set the padding appropriately.